c7n/resources/cw.py

```diff
@@ -255,7 +250,7 @@ class LastWriteDays(Filter):
 
     def process(self, resources, event=None):
         client = local_session(self.manager.session_factory).client('logs')
-        self.date_threshold = datetime.utcnow() - timedelta(
+        self.date_threshold = parse_date(datetime.utcnow()) - timedelta(
```

Comment:
This parse_date use adds no value does it?

Reply:
It creates a timezone-aware datetime object that can be compared to the other one.

Reply:
A move obvious way to do this would be to pass in the UTC timezone info to the datetime now function. This makes it clear that you are making a timezone aware datetime object, whereas the behaviour from the `parse_date` function is more specialized knowledge.
```python
from dateutil.tz import tzutc
# ...
self.date_threshold = datetime.now(tzutc()) - timedelta(
```

Reply:
Well, I see it more that `parse_date()` provides a normalization. Whatever you pass into it you either get a consistent datetime object or None. So in my mind, it's clearer to use it in both cases. Personally I'd love to see us use it everywhere a datetime object is expected just for consistency.

Reply:
i've gone back and forth on this a few times especially when we explicit know the values, but i've come around to parse_date.